ci-operator/step-registry/azure/provision/custom-role/azure-provision-custom-role-commands.sh

```diff
@@ -284,6 +280,26 @@ ${required_permissions}
 """
     fi
 
+    # Starting from 4.19, user-assigned identity created by installer is removed, related permissions are not required any more.
+    if (( ocp_minor_version <=18 && ocp_major_version == 4 )) || [[ "${CLUSTER_TYPE_MIN_PERMISSOIN}" == "UPI" ]]; then
```

Comment:
so for 4.19+ UPI install, still need the permissions, right ?

Reply:
Good questions.
Currently, they are still required for UPI, but I guess managed identity creation can also be removed from [UPI doc](https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/installing_on_azure/user-provisioned-infrastructure#installation-azure-create-resource-group-and-identity_installing-restricted-networks-azure-user-provisioned) and [ARM template](https://github.com/openshift/installer/blob/main/upi/azure/04_bootstrap.json). Let me verify that.

Reply:
Tested in MR https://gitlab.cee.redhat.com/aosqe/flexy-templates/-/merge_requests/2296, UPI installation is passed without managed identity creation and assignment.

I think we can remove this part from UPI official doc and ARM template on 4.19.

@patrickdillon WDYT? If that's ok, I can raise a PR in installer repo for the change.

Reply:
Agreed. We should remove this from UPI. Technically the identity has not been needed for multiple releases, but removing from 4.19+ is sufficient. 

Thanks for bringing this up, this did not occur to me during refinement.